src/main/flight/imu.c

```diff
@@ -559,6 +610,41 @@ int16_t calculateThrottleAngleCorrection(uint8_t throttle_correction_value)
     return lrintf(throttle_correction_value * sin_approx(angle / (900.0f * M_PIf / 2.0f)));
 }
 
+void imuComputeQuaternionFromRPY(quaternionProducts *quatProd, int16_t initialRoll, int16_t initialPitch, int16_t initialYaw)
+{
+    if (initialRoll > 1800) initialRoll -= 3600;
```

Comment:
Please always use a block after branching statements, and always put statements onto new lines. This will improve the maintainability.